### PR TITLE
add filters to projects

### PR DESCRIPTION
**Pull request by sarveshpro**
Todo - 

- [x] merge into 1 single dropdown 'Filters'
- [x] Fix filter function

### CODE REVIEW

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/**](https://planet-webapp-gh74nu1k0-planetapp.vercel.app/) |
|:-:|
|<a href="https://planet-webapp-gh74nu1k0-planetapp.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/f6JC4DdFASBe3gfCy-TPl2BGbZc_mJYpAXctccRBJTGX-bXog8sb-jMlyiWOoY2fNfNN1KciuwQ6u2G_l2fUhw.png">(view full size)</a> |







Commit 1cfba13b9191df6ef8f7aec3aeb655556c060992 (https://planet-webapp-gh74nu1k0-planetapp.vercel.app).

**Comment by norbertschuler** [comment]
The project purpose filter does not yet filter anything, right? And shall there be info boxes for all the (i) buttons?

Bildschirmfoto 2021-09-15 um 08 20 37

**Comment by sarveshpro** [comment]
> The project purpose filter does not yet filter anything, right? And shall there be info boxes for all the (i) buttons?
> 
project purpose and info boxes hidden.

File: src/features/projects/components/projects/Filters.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  return process.env.TENANT === 'salesforce' && ready ? (
+    <div className={styles.filtersContainer}>
+      <div className={styles.filterButtonContainer}>
+        <div
+          onClick={() => setFilterOpen(!filtersOpen)}
+          className={`${styles.filterButton} ${
+            filtersOpen ? styles.selected : ''
+          }`}
+        >
+          <div className={styles.filterButtonText}>{t('donate:Filters')}</div>
```
Comment (norbertschuler) [inline src/features/projects/components/projects/Filters.tsx:90]: Here a resource `donate:Filters` was no created! It only looks ok in English as the resource name is (wrongly) starting an with uppercase character.

**Review by norbertschuler — COMMENTED** [review]
Found a missing resource.

**Pull request merged**